### PR TITLE
Moyo Header

### PR DESCRIPTION
**Pull request by AndriiZhovtko**
- [DEMO LINK](https://AndriiZhovtko.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://AndriiZhovtko.github.io/layout_moyo-header/report/html_report/)

Remarking code with your advice.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,13 +2,88 @@
...
+              class="nav_link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#Photo"
+              class="nav_link"
+            >
```
Comment (loralevitska) [inline src/index.html:73]: this tag can be written in one line provided that there are less than 3 attributes in the tag (it's up to you)

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
+  height: 60px;
+  margin: auto;
+
+}
+
+.nav_link , .is-active {
+  height: 40px;
+  color: #000;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
```
Comment (loralevitska) [inline src/style.css:29]: Remove redundant  default styles

**Review by loralevitska — CHANGES_REQUESTED** [review]
Please redeploy your solution for review. Before re-request review check links on GIT
image

**Comment by AndriiZhovtko** [comment]
[DEMO LINK](https://andriizhovtko.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://andriizhovtko.github.io/layout_moyo-header/report/html_report/)
Remarking code with your advice__2

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Links are not working
image

**Pull request closed without merging**